app/src/main/java/com/example/campusguide/BottomNavigation.kt

```diff
@@ -0,0 +1,70 @@
+package com.example.campusguide
+
+import android.content.Intent
+import com.example.campusguide.directions.ChooseDestinationOptions
+import com.example.campusguide.directions.ChooseOriginOptions
+import com.example.campusguide.location.FusedLocationProvider
+import com.example.campusguide.map.GoogleMapAdapter
+import com.example.campusguide.utils.permissions.Permissions
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import com.google.maps.model.LatLng
+
+class BottomNavigation constructor (
+    private val activity: MapsActivity,
+    private val map: GoogleMapAdapter,
+    private val permissions: Permissions,
+    private val locationProvider: FusedLocationProvider) {
+
+    private lateinit var bottomNavigationView : BottomNavigationView
+
+    fun setupBottomNavigation() {
+        bottomNavigationView = activity.findViewById(R.id.bottom_navigation)
+        bottomNavigationView.selectedItemId = R.id.campus_map
+        bottomNavigationView.setOnNavigationItemSelectedListener { item ->
+            when (item.itemId) {
+                R.id.directions -> {
+                    startNavigation()
+                }
+                R.id.campus_map -> {
+                    centerCameraSGW()
+                }
+                R.id.poi -> {
+                    // points of interest view
+                }
+            }
+            return@setOnNavigationItemSelectedListener true
+        }
+    }
+
+    private fun centerCameraSGW(){
+        val hall = com.google.android.gms.maps.model.LatLng(
+            45.495792, -73.578096
```

Comment:
We should put these coordinates in Constants.kt if they aren't already there

Reply:
Added coordinate constants for SGW and LOY